VideoLocker/res/layout/activity_login.xml

```diff
@@ -59,47 +59,74 @@
                         android:paddingLeft="44dp"
                         android:paddingRight="44dp">
 
-                        <EditText
-                            android:id="@+id/email_et"
-                            style="@style/edit_text_style"
-                            android:hint="@string/email_username"
-                            android:inputType="textEmailAddress"
-                            android:maxLength="100" />
-
-                        <EditText
-                            android:id="@+id/password_et"
-                            style="@style/edit_text_style"
-                            android:layout_marginTop="8dp"
-                            android:hint="@string/password"
-                            android:inputType="textPassword"
-                            android:maxLength="100" />
+                        <android.support.design.widget.TextInputLayout
+                            android:id="@+id/usernameWrapper"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingStart="-4dp"
+                            android:paddingEnd="-4dp"
+                            android:paddingLeft="-4dp"
+                            android:paddingRight="-4dp"
+                            android:textColorHint="@color/grey_text">
+
+                            <EditText
```

Comment:
Well this is nifty.
![screen shot 2016-08-10 at 12 58 01 pm](https://cloud.githubusercontent.com/assets/7101723/17562732/198363dc-5efa-11e6-9a4c-007cb40e312c.png)